src/test/java/products/ProductsTests.java

```diff
@@ -0,0 +1,14 @@
+package products;
+
+import base.BaseTest;
+import org.testng.annotations.Test;
+import salesforce.ui.pages.ProductPage;
+import static configfile.Configuration.dotenv;
+
+public class ProductsTests extends BaseTest {
+
+//    @Test
```

Comment:
delete comment code

Reply:
I have commented this line because of some problems with the checkstyle in github